ec2/spark_ec2.py

```diff
@@ -346,9 +347,15 @@ def get_spark_ami(opts):
 # Returns a tuple of EC2 reservation objects for the master and slaves
 # Fails if there already instances running in the cluster's groups.
 def launch_cluster(conn, opts, cluster_name):
-    if opts.identity_file is None:
+    if opts.identity_file is None or \
+       not os.path.exists(opts.identity_file):
         print >> stderr, "ERROR: Must provide an identity file (-i) for ssh connections."
```

Comment:
Yes, I will fix this.

It will be hang  up at check cluster ssh ready without warnings.